doc/classes/Tween.xml

```diff
@@ -144,6 +144,12 @@
 				[b]Note:[/b] As it results from accumulating frame deltas, the time returned after the [Tween] has finished animating will be slightly greater than the actual [Tween] duration.
 			</description>
 		</method>
+		<method name="has_tweeners" qualifiers="const">
+			<return type="bool" />
+			<description>
+				Returns [code]true[/code] if any [Tweener] has been added and the tween is valid. Useful for avoiding print an error by checking and killing the [Tween] if it has not tweeners before it is stepped by [SceneTree].
```

Comment:
Why does it mention tween validity if it's not checked? (the `valid` variable)
Also the second sentence likely needs rewording, because it's a bit unclear.

Reply:
> Why does it mention tween validity if it's not checked? (the `valid` variable)

This description is changed multiple time, the mention about validity seems redundant now.


> Also the second sentence likely needs rewording, because it's a bit unclear.


To be honest, my English is terrible. Could you provide a suitable description to help me?

Reply:
```suggestion
				Returns [code]true[/code] if any [Tweener] has been added to the tween. Useful when tweeners are added dynamically and the tween can end up empty. Killing an empty tween before it starts will prevent errors.

Reply:
Thanks :)

Reply:
Wait, the validation description is can not removed.
Event if the tween has been added twenners, the vector `tweeners` will be cleared with `valid` variable, too.